pdal/private/MathUtils.hpp

```diff
@@ -63,10 +63,13 @@ typedef std::shared_ptr<PointView> PointViewPtr;
 namespace math
 {
 
-PDAL_DLL PointViewPtr demeanPointView(const PointView& view);
-PDAL_DLL PointViewPtr demeanPointView(const PointView& ,double* centroid);
-PDAL_DLL PointViewPtr transform(const PointView&, double* matrix);
-PDAL_DLL void transformInPlace(PointView&, double* matrix);
+PointViewPtr demeanPointView(const PointView& view);
```

Comment:
Deliberately not exporting these now private functions? I wonder if any plugins have depended on these?

Reply:
This perhaps needs some discussion. Here I'm making the assumption here that someone writing a plugin has the code and can build in private things if they need them. This came up with `rotate`, which I also stuck in here but was used by the I3S/SLPK plugin.

I was just working on making more stuff private, which I think it should be, but it's also not a fight that I want to get into.

Reply:
> it's also not a fight that I want to get into

Nor I. We've been pretty consistent with that approach to private code as of late, so we can continue it here.